### PR TITLE
Corrected PhpDoc blocks

### PR DESCRIPTION
**Pull request by rkgrep**
I know you don't like such kind of PRs, but there is quite big mess in return types and sometimes in arguments (e. g. pluck helper)

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Thses changes are wrong.

**Comment by GrahamCampbell** [comment]
`mixed|null` makes no sense at all. `mixed` includes ALL types alread.

File: src/Illuminate/Auth/Access/Gate.php [hunk 1]
```diff
@@ -275,7 +275,7 @@ protected function callAuthCallback($user, $ability, array $arguments)
      * @param  \Illuminate\Contracts\Auth\Authenticatable  $user
      * @param  string  $ability
      * @param  array  $arguments
-     * @return bool|void
+     * @return bool|null
```
Comment (GrahamCampbell) [inline src/Illuminate/Auth/Access/Gate.php]: this change looks right though - could you PR that one plz

File: src/Illuminate/Mail/Transport/MailgunTransport.php [hunk 2]
```diff
@@ -110,7 +110,7 @@ public function getKey()
      * Set the API key being used by the transport.
      *
      * @param  string  $key
-     * @return void
+     * @return string
```
Comment (GrahamCampbell) [inline src/Illuminate/Mail/Transport/MailgunTransport.php]: i think the return here was unintentional

**Comment by GrahamCampbell** [comment]
Anything that says `mixed|` is wrong. Please could you send this again without those changes.

**Comment by rkgrep** [comment]
I fixed the commit, could you reopen this, please.

**Comment by GrahamCampbell** [comment]
Can't. It's greyed out.

**Pull request closed without merging**